app/views/rating/applications/_application.html.slim

```diff
@@ -18,8 +18,8 @@ tr
   td.text-left = application.flags.map { |f| content_tag(:span, f, class: 'label label-default') }.join('<br>').html_safe
   td.text-center = icon('check') if application.application_data['signed_off_at_project1']
   td.text-center = icon('check') if application.application_data['signed_off_at_project2']
-  td.text-center = icon('check') if application.application_data['mentor_fav_project1']
-  td.text-center = icon('check') if application.application_data['mentor_fav_project2']
+  td.text-center = icon('heart') if application.application_data['mentor_fav_project1'] == 'true'
```

Comment:
Ok. Will take care of merging and adding the respective task tonight. Nice, thanx 🙏